apps/extension/src/core/managers/usage.ts

```diff
@@ -0,0 +1,41 @@
+import { BaseManager } from "./base"
+
+export interface Usage {
+  key: string,
+  used: number
+}
+
+export type UsageData = Pick<Origin, "key">
```

Comment:
`Origin` doesn't seem to be imported

Reply:
I don't see a `key` attr in Origin so not sure what the intent is - assuming this is WIP

Reply:
Yes, this is WIP.